mmv1/products/vertexai/FeatureGroup.yaml

```diff
@@ -90,6 +90,8 @@ properties:
   - !ruby/object:Api::Type::NestedObject
     name: bigQuery
     description: Indicates that features for this group come from BigQuery Table/View. By default treats the source as a sparse time series source, which is required to have an entityId and a feature_timestamp column in the source.
+    update_mask_fields:
+      - 'bigQuery.entityIdColumns'
```

Comment:
Do we have a test that updated this field before? If so, why was it not failing? If not, can you add one?

Reply:
hi @slevenick I've included the update to this field now in the update test. Pls check. I think previously updation of this field wasn't supported